crates/matrix-sdk/src/sliding_sync.rs

```diff
@@ -87,22 +86,14 @@ pub enum RoomListEntry {
 impl RoomListEntry {
     /// Is this entry empty or invalidated?
     pub fn empty_or_invalidated(&self) -> bool {
-        matches! {&self, RoomListEntry::Empty | RoomListEntry::Invalidated(_)}
+        matches! {self, RoomListEntry::Empty | RoomListEntry::Invalidated(_)}
```

Comment:
```suggestion
        matches!(self, RoomListEntry::Empty | RoomListEntry::Invalidated(_))
```

Reply:
when is one preferred over the other?

Reply:
Depends on the macro. `vec!` is basically always used with square brackets, `format!` with round parens. Generally I'd use the same thing [the docs](https://doc.rust-lang.org/std/macro.matches.html) show.